examples/ding-dong-plugin-bot.py

```diff
@@ -0,0 +1,21 @@
+"""doc"""
+import asyncio
+from typing import Optional
+from wechaty import Wechaty
+from python_wechaty_plugin_contrib import (
+    DingDongPlugin, WeatherPlugin
+)
+from python_wechaty_plugin_contrib.ding_dong_rank_plugin import DingDongRankPlugin
+
+bot: Optional[Wechaty] = None
+
+
+async def main():
+    """doc"""
+    # pylint: disable=W0603
+    global bot
+    bot = Wechaty().use([DingDongPlugin(), WeatherPlugin(), DingDongRankPlugin()])
+    # await bot.stop()
+    await bot.start()
```

Comment:
below code is also right: 
```python
bot = Wechaty()
bot.use(DingDongPlugin()).use(WeatherPlugin())
```

Reply:
Can we add support to use `bot.use(DingDong(), Weather())` so that it can be as same as the TS version? Because it is less code and also my favorite.

And I have a suggestion for the Plugin name.

If you pay attention to the wechaty-plugin-contrib in TS, you will see that the exported name is `DingDong`, with the typing `DingDong: (config: Config) => WechatyPlugin`, which means that the `DingDong` is a factory/creator function that returns the `WechatyPlugin`.

So I believe in Python it would be better for us to follow the same naming style: 

```py
from wechaty_plug_contrib import (
  DingDong,
)

DingDongPlugin = DingDong()
wechaty.use(DingDongPlugin)
````

The above code shows that:

1. The wechaty-plugin-contrib module exports `Factory` / `Creator` functions, with the name `DingDong`
1. The plugin factory/creator function `DingDong` will return the `Plugin` when it has been called: `DingDongPlugin = DingDong()`

Does that make sense? If so, let's keep the naming style consistent between the TS and PY.

Reply:
Where does `Facotry`/`Creator` have an advantage over basic class initilization ? 

@huan  In addition, I think: `DingDongPlugin = DingDong()` is weird. 

- `DingDongPlugin` is a instance of the `DingDong`, the name is captial.  Which is not suitable for naming style for python.
- `DingDongPlugin` mean a plugin concept. But `DingDong` doesn't have a `plugin` concept by name.